packages/cli/generators/app/templates/index.js.ejs

```diff
@@ -2,5 +2,8 @@ const application = (module.exports = require('./dist'));
 
 if (require.main === module) {
   // Run the application
-  application.main();
+  application.main().catch(err => {
+    console.log('Cannot start the application.', err);
```

Comment:
We should use `console.error` instead of `console.log` so that the error message goes to `stderr`.